app/components/BreadcrumbNav.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  Breadcrumb,
+  BreadcrumbBar,
+  BreadcrumbLink,
+} from '@trussworks/react-uswds'
+
+export default function BreadcrumbNav({
+  path,
+  pathPrepend,
+  className,
+}: {
+  path: string
+  pathPrepend?: string
+  className?: string
+}) {
+  const [lastBreadcrumb, ...breadcrumbs] = path.split('/').reverse()
+  const sortedCrumbs = breadcrumbs.reverse()
```

Comment:
Ah, I didn't realize that spread syntax requires the element with ellipses to be the last one.

Here's a simpler approach:

```ts
const breadcrumbs = path.split('/')
const lastBreadcrumb = breadcrumbs.pop()
```